android/src/main/java/com/agontuk/RNFusedLocation/RNFusedLocationModule.java

```diff
@@ -167,6 +168,10 @@ private void onLocationSettingsResponse(Task<LocationSettingsResponse> task) {
                      *
                      * TODO: we may want to make it optional & just say that settings are not ok.
                      */
+                    if(!mShowLocationDialog) {
+                        break;
```

Comment:
Before break, we should invoke the error callback to finish the request.

```java
invokeError(LocationError.SETTINGS_NOT_SATISFIED.getValue(),
    "Location settings are not satisfied.");
```

Reply:
Updated.